src/encoded/tests/test_no_attype.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from encoded.tests.features.conftest import app_settings, app, index_workbook
+
+
+pytestmark = [
+    pytest.mark.indexing,
+    pytest.mark.usefixtures('index_workbook'),
+]
+
+
+'''
+    Used to test views with no @type property.
```

Comment:
Should probably be comment instead of docstring, or put docstring into function. https://www.python.org/dev/peps/pep-0257/